complete.js

```diff
@@ -49,7 +49,7 @@ foreach ($pre_reqs as $pre_req) {
 }
 
 /* Send out the translations, too. */
-echo file_get_contents('cc-translations.js.' . $gettextlang);
+echo file_get_contents('translations/cc-translations.js.' . $gettextlang);
```

Comment:
Why are you changing these paths? is there an error with them?

Reply:
yes you can see the error during the php execution of complete.js that the file cannot be found in the web server log